src/ratelimit.ts

```diff
@@ -256,7 +256,10 @@ export abstract class Ratelimit<TContext extends Context> {
     await this.limiter().resetTokens(this.ctx, pattern);
   };
 
-  public getRemaining = async (identifier: string): Promise<number> => {
+  public getRemaining = async (identifier: string): Promise<{
+    remaining: number;
+    reset: number;
```

Comment:
I feel like `reset` doesn't tell much about this field's purpose. Maybe we can call it `resetsAt`

Reply:
I used reset because that's the name we use in [limit](https://upstash.com/docs/oss/sdks/ts/ratelimit/methods#limit)

I think we should comply with the naming in limit

Reply:
Maybe we can write a little JSDOC for `getRemaining` about its inputs and outputs.